irohad/torii/impl/command_service.cpp

```diff
@@ -47,11 +47,18 @@ namespace torii {
         log_(logger::log("CommandService")) {
     // Notifier for all clients
     tx_processor_->transactionNotifier().subscribe([this](auto iroha_response) {
-      // Find response in cache
+      // find response for this tx in cache; if status of received response is
+      // "less" than cached one, dismiss received one
```

Comment:
`isn't "bigger"` (according to the code)

Reply:
Why? line 58, we compare, if status of received is less or equal, we do nothing, and if it's not, we put the new response to cache

Reply:
less<b> or equal</b>